cross-chain/arbitrum/package.json

```
@@ -0,0 +1,75 @@
+{
+  "name": "@keep-network/tbtc-v2-arbitrum",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.0.1-dev",
```
I think on `main` most of the time we should keep the `-dev` suffixed version. The version with no suffix should be used only to create the mainnet package.